### PR TITLE
Fixed typos in README.md

### PR DESCRIPTION
**Pull request by johnzl-777**
## Description of Issue (if Applicable)

Some typos exist in the README.md in the Qiskit Textbook Notebooks section ("add a" instead of just "a" and "thebe" should just be "the")

## What Changes Were Made?

Fixed the typo from "a" to "add" and "thebe" to "the" in the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ You will need to fork the textbook, make changes in your own branch, then submit
 
 ## Tags in Qiskit Textbook Notebooks
 
-After writing your chapter, you must a few tags to your cells (You can see the tags using `view > tags`).
+After writing your chapter, you must add a few tags to your cells (You can see the tags using `view > tags`).
 
 - If you include the chapter's title in your notebook, move this to its own cell and add a `remove-cell` tag to avoid duplicate titles on the website. This is only for the title that will appear in the sidebar.
 - If you intentionally raise an exception in a cell, add a `raises-exception` tag to that code cell so our checking script knows this is intentional.
-- Any code cell that uses IBM hardware, or relies on the results from such a cell needs a `uses-hardware` tag to alert the user this cannot be run on the website through thebe lab.
+- Any code cell that uses IBM hardware, or relies on the results from such a cell needs a `uses-hardware` tag to alert the user this cannot be run on the website through the lab.
```
Comment (frankharkins) [inline README.md:44]: 'thebelab' is actually the name of the software used for interactivity. I suggest changing to either 'thebelab' or maybe linking to the site (https://thebelab.readthedocs.io/en/latest/) to make clear it's not a spelling error.

**Comment by frankharkins** [comment]
Thank you for this contribution. I have made a change, let me know if you approve and I'll merge asap.

File: README.md (hunk 1 above)
Comment (johnzl-777) [reply README.md]: Whoops! Thank you for pointing that out!

**Comment by johnzl-777** [comment]
Looks good! Thank you for looking things over, feel free to merge whenever 👍

**Pull request merged**